config-classic/ClassicPlatformConfigProvider.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.configclassic;
+
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import com.powsybl.commons.config.ModuleConfigRepository;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.config.PlatformConfigProvider;
+import com.powsybl.commons.config.PlatformEnv;
+import com.powsybl.commons.config.EnvironmentModuleConfigRepository;
+import com.powsybl.commons.config.StackedModuleConfigRepository;
+
+import com.google.auto.service.AutoService;
+
+/**
+ * The standard powsybl PlatformConfig provider. It uses System Properties to
+ * get config dirs (powsybl.config.dirs, itools.config.dir; defaults to
+ * $HOME/.itools) and reads configuration from yaml, xml or java properties
+ * files. The config dir names can use the keywords from {@link PlatformEnv}
+ * (e.g. app.root, user.home). It also uses
+ * {@link EnvironmentModuleConfigRepository} to read configuration from
+ * environment variables.
+ *
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@AutoService(PlatformConfigProvider.class)
+public class ClassicPlatformConfigProvider implements PlatformConfigProvider {
+
+    private static final String NAME = "standard";
```

Comment:
Perhaps you should change the name as `classic` here too since you changed the class' name?

Reply:
Done thanks